### ISSUE TITLE
Raw Ores + SMELT

### ISSUE THREAD

**Issue by Pandas0** [GC-spigot/AdvancedEnchantments#2201]
**Describe the bug**
When mining iron, gold, and copper ore on 1.17 it has a range of the amount of 'raw' ore it can give back.

Iron Ore + Gold Ore: (You have a chance to gain 1x more ore per fortune level)
No Fortune: 1x Raw Ore
Fortune 1: 1-2 Raw Ore
Fortune  2: 1-3 Raw Ore
Fortune 3: 1-4 Raw Ore

Copper Ore: (You have have a chance to gain 2-3x more ore per fortune level)
No Fortune: 2-3x Raw Ore
Fortune 1: 2-6x Raw Ore
Fortune 2: 2-9x Raw Ore
Fortune 3: 2-12x Raw Ore

The bug with AE is when you mine Iron + Gold  Ore it gives back 1x Iron/Gold Ingot every time (With any level of Fortune) and when mining Copper Ore it gives back 2x Copper Ingots (With Fortune 0-2) and 4x Copper Ingots (With Fortune 3).

In simple terms, the bug is SMELT doesn't give the correct amount of Ingots to what the Raw ore would of been without the CE. In most cases, not using SMELT would result in more ore.

**To Reproduce**
Load AE on a 1.17 server and get an enchantment that uses SMELT, get 4 pickaxes with Fortune 0-3, mine Iron/Gold Ore and then mine Copper Ore.

**Screenshots / Video**
N/A

## Server Information
- "/ae plinfo" https://paste.md-5.net/umanexarop#L184
- Server log: https://mcpaste.io/a80036b7a131f184
